src/pages/groupComparison/ClinicalData.tsx

```diff
@@ -273,6 +274,31 @@ export default class ClinicalData extends React.Component<
         this.horizontalBars = !this.horizontalBars;
     }
 
+    @action.bound
+    private onClickShowNA() {
+        this.showNA = !this.showNA;
+    }
+
+    private readonly clinicalDataPromiseFiltered = remoteData({
+        await: () => [
+            this.clinicalDataPromise,
+            this.props.store.samples
+        ],
+        invoke: async () => {
+            const axisData = this.clinicalDataPromise.result!;
+            // for string like values include NA values if requested
+            // TODO: implement for numbers
+            if (this.showNA && axisData.datatype === 'string' && this.props.store.samples.result!.length > 0) {
+                const naSamples = _.difference(_.uniq(this.props.store.samples.result!.map(x => x.uniqueSampleKey)), _.uniq(axisData.data.map((x) => x.uniqueSampleKey)));
```

Comment:
i would comment this to make it more clear what's happening.  i'm kinda surprised prettier allows such a long line.  maybe do 
```
_.difference(
   thingA,
   thingB
)

```

Reply:
ah yes prettier actually did catch this - fixed now